cmd/terramate/cli/cloud.go

```diff
@@ -645,6 +667,34 @@ func getGithubPRByCommit(ghClient *github.Client, owner, repo, commit string) (*
 
 }
 
+func getGithubPRReviewDecision(qlClient *githubql.Client, owner, repo string, pullNumber int) (string, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), defaultGithubTimeout)
+	defer cancel()
+
+	var q struct {
+		Repository struct {
+			PullRequest struct {
+				ReviewDecision string
```

Comment:
this can be `null`.

Reply:
What do you mean? The string can be ""? Yes, it can. How would you handle it differently?

Reply:
haha I actually meant the response, sorry.
Here: https://github.com/terramate-io/terramate/pull/1525/files#diff-338c2ed255d3f2d07e303ff13d3302050f7986df605f31cd33fa648c24050078R239